cedar-policy/src/ffi/is_authorized.rs

```diff
@@ -584,8 +586,15 @@ impl AuthorizationCall {
             }
         };
 
-        let (policies, entities) = match self.slice.try_into(schema.as_ref()) {
-            Ok((policies, entities)) => (policies, entities),
+        let entities = match self.entities.parse(schema.as_ref()) {
+            Ok(entities) => entities,
+            Err(e) => {
+                return build_error(vec![e], warnings);
```

Comment:
Do we want to wait and try to parse policies before returning? That way we include more errors